### PR TITLE
Dark mode

### PR DESCRIPTION
**Pull request by j24m**
Hello people I have Implemented dark mode have a look. Kindly review my code.

### CODE REVIEW

**Review by samyakshah3008 — APPROVED** [review]

File: darktheme.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+body {
+    color: var(--secondary-color);
+}
+
+body.dark-theme {
+    color: var(--light-color);
+    background-color: var(--button-floatHover-color);
+}
+
```
Comment (rahulyadav139) [inline darktheme.css:9]: Appreciation: Great work Jyotsna. Dark theme is really looking dope.  Background color (for the whole page) in dark theme adds a punch to the website.

**Review by rahulyadav139 — APPROVED** [review]

File: cart.html [hunk 2]
```diff
@@ -7,10 +7,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>FlameKeys | Cart Page </title>
     <script src="https://kit.fontawesome.com/d3026d29a6.js" crossorigin="anonymous"></script>
-
+    <link rel="stylesheet" href="./darktheme.css">
     <link rel="stylesheet" href="./main.css">
     <link rel="stylesheet" href="https://flameui.netlify.app/components.css">
```
Comment (zoef-yt) [inline cart.html:12]: Suggestion: Make sure you add this all above the other this will help you override CSS easily

File: main.css [hunk 3]
```diff
@@ -3,18 +3,21 @@
...
     font-size: 62.5%;
     font-family: "Roboto Condensed", sans-serif;
 }
-:root{
+
+:root {
     --footer-icon-hover-color: darkslateblue;
-    --link-hover-bg-color:#eaece5;
+    --link-hover-bg-color: #eaece5;
+    --dark-theme-box-shadow: 10px 10px 20px 2px rgba(0, 0, 0, 1);
```
Comment (zoef-yt) [inline main.css:20]: Suggestion: stay consistent with your colour type, I see you used hex for your colour so use hex here too.

**Review by zoef-yt — APPROVED** [review]
The UI looks absolutely amazing. The dark mode is just love. Great code do the suggested. Nice use of comments everywhere. This pr is good to merge. As this was a visual change adding a screenshot would've been good. Great job Jyotsna.

File: main.css (hunk 3 above)
Comment (j24m) [reply main.css:20]: I did this in a hurry, will make the changes soon, thanks for noticing. :)

**Pull request merged**